gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -476,6 +479,12 @@
   public static final String DEFAULT_JOB_HISTORY_STORE_PASSWORD = "gobblin";
 
   /**
+   * Password encryption and decryption properties.
+   */
+  public static final String ENCRYPT_KEY_FS_URI = "encrypt.key.fs.uri";
```

Comment:
You can use Path(URI aUri) and then Path.getFileSystem() to get the file system associated with the URI.